variant-caller/variant-caller-wdl-checker/topmed-variantcaller-checker.wdl

```diff
@@ -14,72 +16,13 @@ task checkerTask {
 
 
   command <<<
```

Comment:
I feel like it would be better to call: `python3.5 utils/run_concordance_docker.py` than to duplicate the code above.